src/main/proto/ga4gh/read_service.proto

```diff
@@ -91,6 +99,49 @@ message GetReadGroupSetRequest {
   string read_group_set_id = 1;
 }
 
+// ******************  /readgroups  *********************/
+
+// This request maps to the body of `POST /readgroups/search` as JSON.
+
+
+message SearchReadGroupsRequest {
+  // The dataset to search (required).
+  string dataset_id = 1;
```

Comment:
I think  we should add back in ``read_group_set_id`` (optional). I'm thinking of the case here where we need to find the ReadGroups that correspond to a particular ReadGroupSet so that the client can run a ``SearchReadsRequest``. With this inferface, the client will have to get back all of the ReadGroups in a dataset (potentially hundreds of thousands), and then filter for the read group set they want. 

I guess the alternative to this is to add a ``read_group_set`` option to ``SearchReadsRequest`` to cover this use-case. Since Google already does this (AFAIK), this might be the simpler option.